doc/whats_new/v0.23.rst

```diff
@@ -118,6 +118,15 @@ Changelog
   when `y=None`.
   :pr: `15918` by :user: `Luca Kubin <lkubin>`.
 
+:mod:`sklearn.neighbors`
+..............................
+
+- |Fix| Fixed bug in function _radius_neighbors_from_graph(), module 
```

Comment:
Maybe we should mention that the issue was to get N-D array instead of 1-D array:

```
- |Fix| fix a bug which was converted list of array into 2-D object
  array instead of 1-D array containing NumPy array. This bug
  was affecting :meth:`neighbors.NearestNeighbors.radius_neighbors`.
```